src/PrestaShopBundle/Resources/views/Admin/Module/Includes/action_menu.html.twig

```diff
@@ -44,12 +44,10 @@
           data-confirm_modal="module-modal-confirm-{{ name }}-{{ url_active }}">
           {{ url_active|capitalize|replace({'_': " "})|trans({}, 'Admin.Actions') }}
       </button>
-      {% if urls|length > 1 %}
-      <input type="hidden" class="btn">
```

Comment:
for history, what do we need this hidden button?

Reply:
This is the trick. In order to get the bootstrap design with a button near a dropdown, we need two `btn` side by side.
You can see that without it, I would have a `<form>` on which I cannot apply the `btn` class.

Reply:
I still don't understand the reason for the hidden button

Reply:
Well, remove it and you will see.

Reply:
Oh I see now, we have one form per button, and that breaks layout because split buttons need to be side by side.

This needs to be refactored later using `formaction` so that we have only one form per module.

Reply:
We have a form per action per module, because we send them as POST.
We put the maximum code as possible in the template in order to make it work with or without javascript.

Reply:
`formaction` should work fine with POST and without javascript. I tried a quick fix but it didn't  work. I think the confirmation box was messing something up, but I didn't have time to figure out the source of the problem, so I left it be.

Regarding the need to make it work without javascript... I think it's useless doing that in the BO.